Tools/LyTestTools/tests/integ/sanity_tests.py

```diff
@@ -15,40 +15,51 @@
 import ly_test_tools.environment.process_utils as process_utils
 import ly_test_tools.environment.waiter as waiter
 
-pytestmark = pytest.mark.SUITE_smoke
-
 logger = logging.getLogger(__name__)
 
 # Note: For device testing, device ids must exist in ~/ly_test_tools/devices.ini, see README.txt for more info.
 
 
 @pytest.mark.parametrize("project", ["AutomatedTesting"])
 class TestAutomatedTestingProject(object):
+
     def test_StartGameLauncher_Sanity(self, project):
+        # Kill processes that may interfere with the test
         process_utils.kill_processes_named(names=process_utils.LY_PROCESS_KILL_LIST, ignore_extensions=True)
 
         try:
+            # Create the Workspace object
             workspace = helpers.create_builtin_workspace(project=project)
 
+            # Create the Launcher object and add args
             launcher = launcher_helper.create_launcher(workspace)
-            launcher.args.extend(['-NullRenderer', '-BatchMode'])
+            launcher.args.extend(['-NullRenderer'])
```

Comment:
Believe the arg for null renderer is now -rhi=Null. This would likely fail on Jenkins.

Reply:
fixed